CorsixTH/Lua/dialogs/resizables/file_browsers/save_map.lua

```diff
@@ -63,6 +63,14 @@ function UISaveMap:abortName()
   self.new_map_textbox.panel:setLabel(_S.save_map_window.new_map)
 end
 
+--! Updates the textbox to selected file
+--!param label (string) Selected file name
+function UISaveMap:updateTextbox(label)
```

Comment:
`updateDisplayedFilename` or so?

Say what the intent is instead, the text-box is just the means to do it.

Compare with: Hammering a nail is not about hammering a nail, but about connecting some wood to each other for some purpose. The former is the means, the latter is the intent.